assets/js/hooks/useMonitorInternetConnection.js

```diff
@@ -53,15 +54,9 @@ export function useMonitorInternetConnection() {
 		}
 
 		try {
-			const connectionCheckResponse = await apiFetch( {
-				path: '/google-site-kit/v1/',
-			} );
-
-			// We are only interested if the request was successful, to
-			// confirm online status.
-			const canReachConnectionCheck = !! connectionCheckResponse;
+			await fetch( `${ rootURL }google-site-kit/v1/` );
```

Comment:
Let's use `URL` to combine the relative path with the base safely. From what I've seen it seems to always include a trailing slash, but it is a filterable value so better to not assume.

```suggestion
			await fetch( new URL( '/google-site-kit/v1/', rootURL ) );
```